code/modules/mob/living/silicon/robot/robot_modules.dm

```diff
@@ -266,15 +263,15 @@
 		/obj/item/scalpel,
 		/obj/item/circular_saw,
 		/obj/item/bonesetter,
-		/obj/item/extinguisher/mini,
 		/obj/item/roller/robo,
 		/obj/item/borg/cyborghug/medical,
```

Comment:
I DEMAND that you give every borg a hugging module

Reply:
giving them staves of healing would also be nice, just so that the mediborg doesn't outclass them TOO badly